box-ui/src/components/css/BoxMeeting.css

```diff
@@ -29,3 +29,61 @@
 .JitsiComponent {
     flex-grow: 3;
 }
+
+.containerStudent {
+    display: flex;
+    flex-wrap: nowrap;
+    background-color: black;
+}
+
+.sectionButtonsStudent {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background-color: black;
+    align-items: center;
+}
+
+.buttonStudent {
+    color: white;
+    background-color: #141414;
+    border-radius: 8px;
```

Comment:
For better responsiveness you should use `rem` and `em` unit as they scale with the user font size